scripts/newAlias.ts

```diff
@@ -0,0 +1,12 @@
+const aliases = require('../spaces/aliases.json');
+const https = require('https');
+
+const args = process.argv.slice(2);
+if (args.length === 2) {
+  const oldSpace = args[0];
+  const newSpace = args[1];
```

Comment:
```suggestion
  const newSpace = args[1];
  
  if (!oldSpace || !newSpace.endsWith('.eth')) {
    console.error('Invalid arguments')
    process.exit(1);
  }
  
```

Let's also exit early when argument are not valid